packages/expo-modules-core/common/cpp/EventEmitter.cpp

```diff
@@ -2,6 +2,8 @@
 #include "EventEmitter.h"
 #include "LazyObject.h"
 
+#include <cxxreact/ErrorUtils.h>
```

Comment:
nit: little afraid if this import will break use_frameworks. could you help to check it?

Reply:
checking on sdk 51 blank project and it should be fine.